CMP1903_A1_2324/Game.cs

```diff
@@ -4,18 +4,21 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace CMP1903_A1_2324
-{
-    internal class Game
-    {
-        /*
-         * The Game class should create three die objects, roll them, sum and report the total of the three dice rolls.
-         *
-         * EXTRA: For extra requirements (these aren't required though), the dice rolls could be managed so that the
-         * rolls could be continous, and the totals and other statistics could be summarised for example.
-         */
-
-        //Methods
+namespace CMP1903_A1_2324 {
+  internal class Game {
+    //Methods
+    public List<int> RollDie() { //New method that returns a list of integers
+      List<int> dieResults = new List<int>(); //Create a new list of integers
+      Die die; //Create a die variable
 
+      //Rolling the die 3 times
+      for (int i = 0; i < 3; i++) {
+        die = new Die(); //Create a new die object
+        die.DieValue = die.Roll(); //Roll the die, assign the result to the DieValue property
+        dieResults.Add(die.DieValue); //Add the die value to the list
+      }
+      
+      return dieResults; //Return the list
```

Comment:
Mandatory requirements have been met

Optional requirements such as reporting some stats on the rolls and allowing for multiple rolls to happen needs to be added for a first, which i recommend adding as think could be a first piece of code.

Formatting is on the ball to the style guide

readability is done well again with no chance of errors